barretenberg/cpp/src/barretenberg/ultra_honk/mega_honk.test.cpp

```diff
@@ -390,7 +396,12 @@ TYPED_TEST(MegaHonkTests, PolySwap)
     auto proving_key_2 = std::make_shared<typename TestFixture::DeciderProvingKey>(builder_copy, trace_settings);
 
     // Tamper with the polys of pkey 1 in such a way that verification should fail
-    proving_key_1->proving_key.polynomials.w_l.at(5) = 10;
+    for (size_t i = 0; i < proving_key_1->proving_key.circuit_size; ++i) {
```

Comment:
this didn't need to change, I just wanted to make sure that tampering was a bit more robust. In theory setting a wire value at a row where no gates/copy constraints are active could still (correctly) lead to a verifiable proof